app/controllers/media_reserves_controller.old.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
```

Comment:
I don't think we need a controller or model for MediaReserve — I thought we were only going to have a ChangeSet

Reply:
I meant to remove this.

Reply:
The reason I added the model and mapped the media_reserves resource to look the scanned_resources controller in the routes was because I couldn't add an 'New Media Reserve' button in the admin controls (+symbol). But maybe I can map the name only (the second'New Scanned Resource' ) to a new alias 'New Media Reserve'. looking into that. thanks